ide/static/js/content.js

```diff
@@ -92,8 +93,28 @@ class Content extends React.Component {
   }
   addNewLayer(layer) {
     const net = this.state.net;
-    net[`l${this.state.nextLayerId}`] = layer;
-    this.setState({ net, nextLayerId: this.state.nextLayerId + 1 });
+    const layerId = `l${this.state.nextLayerId}`;
+    var totalParameters = this.state.totalParameters;
+    net[layerId] = layer;
+    // Parsing for integer parameters when new layers are added as by default all params are string
```

Comment:
Also add a comment that some parameters might have been missed, or are you sure you've covered all?